src/common/valgrind_internal.h

```diff
@@ -135,9 +135,9 @@ extern unsigned _On_valgrind;
 
 #include "valgrind/pmemcheck.h"
 
-extern void util_emit_log(const char *lib, const char *func, int order);
-
+void _pobj_emit_log(const char *func, int order);
```

Comment:
C standard says: "All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use."